lib/foreman_plugin_template/engine.rb

```diff
@@ -16,11 +16,16 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_plugin_template.register_plugin', :before => :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_plugin_template do
-        requires_foreman '>= 1.16'
+        requires_foreman '>= 2.0.0'
```

Comment:
We will need to bump the version if we want to use the core router.
```suggestion
        requires_foreman '>= 2.3.0'
```